.github/workflows/ruby.yml

```diff
@@ -6,13 +6,16 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.6', '3.0' ]
 
     steps:
     - uses: actions/checkout@master
-    - name: Set up Ruby 2.6
-      uses: actions/setup-ruby@v1
+    - name: Set up Ruby ${{ matrix.ruby }}
+      uses: ruby/setup-ruby@fdcfbcf14ec9672f6f615cb9589a1bc5dd69d262
```

Comment:
This is the same SHA that is used in a few places in `github/github`.

Reply:
:+1: this is only a few commits old, and the newer commits don't seem to be worth bringing in.